source/jormungandr/jormungandr/authentication.py

```diff
@@ -114,6 +114,16 @@ def get_token():
         return auth
 
 
+def can_read_user():
+    try:
+        User.can_read()
+    except Exception as e:
+        logging.getLogger(__name__).error('No access to table User (error: {})'.format(e))
+        g.can_connect_to_database = False
+        return False
+    return True
```

Comment:
User.query.first() -> Done
g.can_connect_to_database = True -> There is no use of this positive value since the value will be valid for only one query to navitia..